### ISSUE TITLE
Duda productos y numero de cajas

### ISSUE THREAD

**Issue by ignaciomanzanares** [IIC2413/Syllabus-2023-1#197]
Tengo una duda, en la tabla de productos hay algunos con el atributo num_cajas 10 por ejemplo pero en la tabla cajas los ids van desde 0 a 4 entonces no se puede tener un producto en 10 cajas. Estos productos los borro o que deberia hacer?

**Comment by NicolasOlmosQuiroga** [comment]
¡Hola!, como se menciono en una issue anterior, ustedes deciden el como corregir eso, pero tienen que dejar claro en su entrega el como trabajaron y/o modificaron los datos. Un ejemplo seria re-calcular el numero_cajas con lo que aparece en la tabla cajas.

Saludos.

**Comment by ignaciomanzanares** [comment]
Ok pero lo que no puede pasar es que hayan dos cajas con mismo id y mismo id_producto verdad?

**Comment by NicolasOlmosQuiroga** [comment]
Exacto, recuerda que se deben borrar los datos repetidos (cuidado que algunos puede que tengan solo 1 de los atributos diferentes, y eso implica que no son los mismos datos).

Saludos.

**Comment by ignaciomanzanares** [comment]
Gracias!
